Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -592,8 +595,8 @@ GRefPtr<GstCaps> capsFromSDPMedia(const GstSDPMedia* media)
             gst_structure_remove_fields(structure, "a-setup", "a-ice-ufrag", "a-ice-pwd", "a-sendrecv", "a-inactive",
                 "a-sendonly", "a-recvonly", "a-end-of-candidates", nullptr);
 
-            if (const char* name = gst_structure_get_string(structure, "encoding-name")) {
-                auto encodingName = String(span(name)).convertToASCIIUppercase();
+            if (auto name = gstStructureGetString(structure, "encoding-name"_s)) {
+                auto encodingName = makeString(name).convertToASCIIUppercase();
```

Comment:
You don't need to convert it to String first, there's StringView::convertToASCIIUppercase .